test/run-e2e-ocp.sh

```diff
@@ -22,25 +22,26 @@ declare SHOW_USAGE=false
 
 cleanup() {
 	# NOTE: clears trap INT
-	trap - INT
+	info "Cleaning up ocp..."
 
 	# skip cleanup if user requested help
 	$SHOW_USAGE && return 0
 	delete_obo
+	return 1
```

Comment:
why are we returning 1 ? `return 0` mean everything worked as normal. Any non-zero indicates error

Reply:
Usually we interrupt a program run when found sth abnormal